README.md

```diff
@@ -16,7 +16,68 @@ That's it! IQKeyboardManager takes care of all initialization when your app star
 
 ## Advanced usage
 
-Want to tweak the appearance and/or behavior?
+### Grouping related textfields (previous / next buttons)
+If your UI layout has sibling text fields, then IQKeyboardManager is able to automatically
+add previous / next buttons to the accessorybar which the user can use to jump back and forth.
+See those < and > buttons in the video above.
+
+In case those fields were not direct siblings, until version 1.3.0 of this plugin, you had no way
+to force the previous / next buttons to appear. However, now you can:
+
+#### NativeScript /w XML usage
+Note that for example sake superfluous `<StackLayout>`s were added:
```

Comment:
I think this bit could be expanded on so the reader really understands what’s going on here.

---

Note in the example below that the two `<TextField>` controls are not siblings (both have parent `<StackLayout>` containers). Because of this, IQKeyboardManager will not automatically provide an optimized keyboard by default.

However, if you surround the controls with this plugin’s `<PreviousNextView>` control, as the example below shows, you will continue to get an optimized keyboard as expected.

Reply:
Great!